ui-core/src/components/inputs/InputWithSuggestions.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {
+  useState,
+  useRef,
+  useEffect,
+  ChangeEventHandler,
+  KeyboardEventHandler,
+  ReactNode,
+  FocusEventHandler,
+} from 'react';
+import cx from 'classnames';
+import Input, { InputProps } from './Input';
+
+interface InputWithSuggestionsProps<T> extends Omit<InputProps, 'onChange' | 'value'> {
+  suggestions: Array<T>;
+  onChange: (value: T) => void;
+  getSuggestionLabel: (option: T) => string;
+  customLabel?: ReactNode;
+  numberOfSuggestionsToShow?: number;
+  exactSearch?: boolean;
```

Comment:
So what about your discussion with @thibautsailly about that ? Did he say we might sometimes want to limit the input with an "exact" search ? If no, we should remove this prop I think.

Reply:
Yes it was a suggestion of @thibautsailly :)